static/scss/layout.scss

```diff
@@ -37,6 +37,7 @@
 @import "final-exam-card";
 @import "staff-learner-info-card";
 @import "automatic-emails";
+@import "learners-card";
```

Comment:
could you put the learners-card scss file in the dashboard subdirectory? we have so much styling for the dashboard, I think it's a little cleaner to keep it all in one directory. there's a file in there (`static/scss/dashboard/dashboard.scss`) which imports all the other dashboard files as well.